src/library/rdt/src/rdt.c

```diff
@@ -39,6 +39,8 @@ SEXP Rdt(SEXP tracer, SEXP rho, SEXP options) {
         handler = setup_promise_tracing(options);
     } else if (!strcmp("debug", sys)) {
         handler = setup_debug_tracing(options);
+    } else if (!strcmp("specialsxp", sys)) {
```

Comment:
Shouldn't there be a convention that given some `tracer_name` the `Rdt()` looks for a function called `setup_tracer_name_tracing()`? This way one can built a tracer, load it using `dyn.load` and use it.